listenbrainz/webserver/static/css/theme/boostrap/modals.less

```diff
@@ -97,6 +97,55 @@
 .modal-body {
   position: relative;
   padding: @modal-inner-padding;
+
```

Comment:
All this new CSS code should live in a new separate file in the root of the css directory, rather than modifying this file in theme/bootstrap.